apollo-portal/src/main/resources/static/index.html

```diff
@@ -36,37 +36,221 @@
     <apollonav></apollonav>
 
     <div id="app-list" class="hidden" ng-controller="IndexController">
+        <div class="left-bar">
+            <div class="app-list-show" ng-click="changeContent('1')" ng-show="whichContent==='1'">
+                <h5>{{'Index.MyProject' | translate }}</h5>
+            </div>
+            <div class="app-list-choose" ng-click="changeContent('1')" ng-show="whichContent!=='1'">
+                <h5>{{'Index.MyProject' | translate }}</h5>
+            </div>
+
+            <div class="favorite-list-show" ng-click="changeContent('2')" ng-show="whichContent==='2'">
+                <h5>{{'Index.FavoriteItems' | translate }}</h5>
+            </div>
+            <div class="favorite-list-choose" ng-click="changeContent('2')" ng-show="whichContent!=='2'">
+                <h5>{{'Index.FavoriteItems' | translate }}</h5>
+            </div>
+
+            <div class="public-namespace-list-show" ng-click="changeContent('3')" ng-show="whichContent==='3'">
+                <h5>{{'Index.PublicNamespace' | translate }}</h5>
+            </div>
+            <div class="public-namespace-list-choose" ng-click="changeContent('3')" ng-show="whichContent!=='3'">
+                <h5>{{'Index.PublicNamespace' | translate }}</h5>
+            </div>
 
-        <section class="media create-app-list">
+            <div class="visited-list-show" ng-click="changeContent('4')" ng-show="visitedApps && visitedApps.length  && whichContent==='4'">
+                <h5>{{'Index.RecentlyViewedItems' | translate }}</h5>
+            </div>
+            <div class="visited-list-choose" ng-click="changeContent('4')" ng-show="visitedApps && visitedApps.length  && whichContent!=='4'">
+                <h5>{{'Index.RecentlyViewedItems' | translate }}</h5>
+            </div>
+        </div>
+        <div class="main-table">
+            <div class="create-app-list" ng-show="whichContent==='1'">
+                <div style="width: 180px;margin-left: 30px" ng-click="goToCreateAppPage()"
+                     ng-if="hasCreateApplicationPermission">
+                    <div href="#" class="thumbnail create-btn hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img src="img/plus-white.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.CreateProject' | translate }}</h5></div>
+                    </div>
+                </div>
+                <div class="display-table" ng-show="createdApps.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 20%">{{'Index.appTable.appId' | translate }}</th>
```

Comment:
I think it's better display the information as those in the project page: App Id, App Name, Department, App Owner, Email. And we could reuse the existing labels, e.g. Common.AppId.

![image](https://user-images.githubusercontent.com/837658/129285101-e50b560d-005c-415c-9b2e-147554b07c53.png)